src/src/PluginStructs/P131_data_struct.cpp

```diff
@@ -113,27 +113,29 @@ bool P131_data_struct::plugin_init(struct EventStruct *event) {
                                                       true,
                                                       _textBackFill);
 
-    gfxHelper->setRotation(_rotation);
-    matrix->setBrightness(std::min(_maxbright, _brightness)); // Set brightness, so we don't get blinded by the light
-    matrix->fillScreen(_bgcolor);                             // fill screen with black color
+    success = (nullptr != gfxHelper);
 
-    # ifdef P131_SHOW_SPLASH
+    if (success) {
+      gfxHelper->setRotation(_rotation);
+      matrix->setBrightness(std::min(_maxbright, _brightness)); // Set brightness, so we don't get blinded by the light
+      matrix->fillScreen(_bgcolor);                             // fill screen with black color
 
-    if (P131_CONFIG_FLAG_GET_SHOW_SPLASH) {
-      uint16_t yPos = 0;
-      gfxHelper->printText(String(F("ESPEasy")).c_str(), 0, yPos, 1, ADAGFX_WHITE, ADAGFX_BLACK);
-      matrix->show();
-      delay(100); // Splash
-    }
-    # endif // ifdef P131_SHOW_SPLASH
+      # ifdef P131_SHOW_SPLASH
 
-    matrix->setTextColor(_fgcolor, _bgcolor); // set text color to white and black background
-    gfxHelper->setColumnRowMode(false);       // Pixel-mode
-    matrix->setTextSize(_fontscaling);        // Handles 0 properly, text size, default 1 = very small
-    matrix->setCursor(0, 0);                  // move cursor to position (0, 0) pixel
-    updateFontMetrics();
+      if (P131_CONFIG_FLAG_GET_SHOW_SPLASH) {
+        uint16_t yPos = 0;
+        gfxHelper->printText(String(F("ESPEasy")).c_str(), 0, yPos, 1, ADAGFX_WHITE, ADAGFX_BLACK);
+        matrix->show();
+        delay(100); // Splash
```

Comment:
Is this really needed, the delay?

Reply:
I'll think of another way to show that splash for a short while

Reply:
Fixed the splash